src/three-components/ARRenderer.js

```diff
@@ -299,19 +342,23 @@ height: 100%;`);
       return;
     }
 
-    for (const view of frame.views) {
-      const viewport = session.baseLayer.getViewport(view);
-      this.renderer.setViewport(0, 0, viewport.width, viewport.height);
-      this.renderer.setSize(viewport.width, viewport.height, false);
-      this.camera.projectionMatrix.fromArray(view.projectionMatrix);
-      const viewMatrix = matrix4.fromArray(pose.getViewMatrix(view));
-
-      this.camera.matrix.getInverse(viewMatrix);
-      this.camera.updateMatrixWorld(true);
-      // NOTE: Clearing depth caused issues on Samsung devices
-      // @see https://github.com/googlecodelabs/ar-with-webxr/issues/8
-      // this.renderer.clearDepth();
-      this.renderer.render(this.scene, this.camera);
-    }
+    const view = frame.views[0];
```

Comment:
What is the rationale behind this change? It seemed like it was important to handle all of the views. Is this no longer the case?

Reply:
The rational was for smartphone AR being monoscopic, but that falls apart for things like Hololens. Will revert, good catch!